xla/service/gpu/model/hlo_op_profiles_data.h

```diff
@@ -741,8 +741,8 @@ constexpr char kDeviceHloOpProfiles[] = R"pb(
         clock_cycles: 15
       }
     }
-  }
-
+  })pb"
```

Comment:
@ddunl fyi I pushed the changes to this file after you approved. I realized they were left out. They are meant to address this error in MSVC:

```
error C2026: string too big, trailing characters truncated
```

Reply:
`clang-format` made the diff massive but it's just breaking it down to a few concatenated strings as opposed to a single very long one.